metrics/report/example.toml

```diff
@@ -0,0 +1,23 @@
+# [mail] section specifies information for
+# a SMTP server authentication as well as the
+# sender and receipts of the report email.
+# This is in order to send the obtained results
+# from checktool to an email whitelist.
+[mail]
+smtp = "myserver.smtp.com"
+port = "25"
+user = ""
+identity = ""
+pass = ""
+from = "pnp@clear-containers.com"
+subject = "PnP failure Report <no-reply>"
+to = [ "manteiner@example.com",
+	"owner@example.com" ]
+cc = [ "contributor@example.com",
+	"developer@example.com" ]
+
+# [checktool] section specifies the arguments for
+# Checktool execution.
+[checktool]
```

Comment:
Given you are encoding the arguments to the tool, you could go one more step in flexibility and encode the path/name of the command as well.
I'll note the tool in question right now is called `checkmetrics` - you may or may not wish to name the TOML section appropriately.
Of course, if you make this a generic 'tool and argument' section, you will need to add the actual argument commands (such as `--metricsdir`) to the TOML as well.